### PR TITLE
feature: hotel details gallery

### PR DESCRIPTION
**Pull request by dave-ok**
https://user-images.githubusercontent.com/19638455/187676837-cd4883b5-f140-4c0a-8886-49dc1385b78a.mp4

### CODE REVIEW

**Review by silksil — DISMISSED** [review]

**Comment by bobetbat** [comment]
please use template components they have a theme and style compatibility. https://minimals.cc/components/mui/dialog/  Scrolling Content Dialogs

File: src/pages/Facility.tsx [hunk 1]
```diff
@@ -30,6 +31,11 @@ export const Facility = () => {
     return createSearchParams(params);
   }, [latestQueryParams, createSearchParams]);
 
+  const FacilityLayout = styled(Box)(({ theme }) => ({
+    paddingLeft: theme.spacing(8),
+    paddingRight: theme.spacing(8)
+  }));
```
Comment (bobetbat) [inline src/pages/Facility.tsx:37]: I would use container to wrap, on small screens padding: 8 could break view, please use implementation as on home page <Container maxWidth='lg'>. Fyi padding should be passed as <MyElement p={8}> . px,py, pt,pb, p and etc

**Review by bobetbat — APPROVED** [review]
LGTM

File: src/pages/Facility.tsx (hunk 1 above)
Comment (dave-ok) [reply src/pages/Facility.tsx:37]: I added the padding during a call with Eli yesterday as we tweaked the designs on the spot per her recommendations.
She wanted this on all pages, but I didn't want to break anything at the moment and felt this should be 
discussed with the team first, so I applied to this page only.

Also `px` does not work in `styled`. I tried it

**Pull request merged**